.github/workflows/CI.yml

```diff
@@ -102,7 +102,7 @@ jobs:
           echo "LIBPQJL_DATABASE_USER=$USER" >> $GITHUB_ENV
         if: ${{ runner.os == 'macOS' }}
       - name: Start Homebrew PostgreSQL service
-        run: pg_ctl -D /usr/local/var/postgres start
+        run: pg_ctl -D /usr/local/var/postgresql@$(psql --version | cut -f3 -d' ' | cut -f1 -d.) start
```

Comment:
Yikes, that's an ugly output format, but I guess this technically works for now. Could we just test on a specific postgres version for macos?

Reply:
Unfortunately GitHub Actions installs only the latest version of PostgreSQL and changes it at will, so we can't depend on a specific version being available. We could install a specific version using brew, but I think relying on brew's install would be more fragile than this. And installing from .dmg would also be risky I think.

I think ideally we would have a PostgreSQL_jll.jl but I'm not sure how best to reconcile that with LibPQ_jll.jl. I bet it could be done and the BinaryBuilder folks would have a good idea how.

For now I think this is the best compromise.